.coafile

```diff
@@ -4,6 +4,10 @@ bears = SpaceConsistencyBear, LineLengthBear, CoffeeLintBear
 use_spaces = True
 max_line_length = 80
 
+[PYTHON]
+files = **.py
+bears = PEP8Bear
+
```

Comment:
Shouldn't we add the deault action :
`default_actions = PEP8Bear: ApplyPatchAction` 
similar to what we've in the main coala repository